languages/ruby/exercises/concept/ostruct/.meta/design.md

```diff
@@ -0,0 +1,15 @@
+## Learning objectives
+
+- Understand how to require from stdlib
```

Comment:
The abbreviation is defined in other place, if accepted.  Still that may not be immediately seen, perhaps it is worthwhile using full words here (we don't tend to use abbreviations in the rest of this document.)

Reply:
```suggestion
- Understand how to require from Standard Library
```